### PR TITLE
Add has experment fn

### PR DESCRIPTION
**Pull request by zshao-aa**
Add `hasExperment` checking

### CODE REVIEW

File: packages/react-ab-testing/src/index.tsx [hunk 1]
```diff
@@ -25,3 +29,8 @@ export function useCohortOf(experimentName: string): string {
     const experiments = React.useContext(ABTestingContext);
     return experiments.getCohort(experimentName);
 }
+
+export function useHasExperiment(experimentName: string): boolean {
```
Comment (SBoudrias) [inline packages/react-ab-testing/src/index.tsx]: I think it'd be cleaner to expose as:

```js
const useABTestingController = () => React.useContext(ABTestingContext)
```

**Review by SBoudrias — APPROVED** [review]

File: packages/ab-testing/src/index.ts [hunk 2]
```diff
@@ -84,6 +84,8 @@ export class Experiments {
         }
         return this.matchedCohorts[experimentName];
     };
+
+    hasExperiment = (experimentName: string): boolean => !!this.config[experimentName];
```
Comment (ZigZagT) [inline packages/ab-testing/src/index.ts:88]: you forgot to add flow type for this one

File: packages/react-ab-testing/src/index.js.flow [hunk 3]
```diff
@@ -1,7 +1,7 @@
 //@flow
 import * as React from 'react';
 import { type ABTestingConfig } from '@appannie/ab-testing';
-export type { ABTestingConfig } from '@appannie/ab-testing';
+export type { ABTestingConfig, ABTestingContextType } from '@appannie/ab-testing';
```
Comment (ZigZagT) [inline packages/react-ab-testing/src/index.js.flow:4]: There's no `ABTestingContextType` definition in `@appannie/ab-testing`

**Comment by ZigZagT** [comment]
@zshao-aa  I think the new change is breaking flow type (see comments above). I might be wrong, can you test it?

**Comment by ZigZagT** [comment]
fixed in https://github.com/appannie/ab-testing/pull/33

**Pull request merged**